cmd/deploy/endpoints.go

```diff
@@ -45,12 +46,31 @@ func Endpoints(ctx context.Context) *cobra.Command {
 		Use:   "endpoints",
 		Short: "Show endpoints for an environment",
 		RunE: func(cmd *cobra.Command, args []string) error {
-
-			if err := contextCMD.LoadManifestV2WithContext(ctx, options.Namespace, options.K8sContext, options.ManifestPath); err != nil {
-				if err := contextCMD.NewContextCommand().Run(ctx, &contextCMD.ContextOptions{Namespace: options.Namespace}); err != nil {
-					return err
+			ctxResource, err := utils.LoadManifestContext(options.ManifestPath)
+			if err != nil {
+				if oktetoErrors.IsNotExist(err) {
+					ctxResource = &model.ContextResource{}
 				}
 			}
+
+			if err := ctxResource.UpdateNamespace(options.Namespace); err != nil {
+				return err
+			}
+
+			if err := ctxResource.UpdateContext(options.K8sContext); err != nil {
+				return err
+			}
+
+			ctxOptions := &contextCMD.ContextOptions{
+				Context:   ctxResource.Context,
+				Namespace: ctxResource.Namespace,
+			}
+			if options.Output == "" {
```

Comment:
what `""` means here?

Reply:
what's the full output with `json` and `md`?

Reply:
"" is the default one, the three outputs are:
<img width="645" alt="Captura de pantalla 2022-04-04 a las 13 55 11" src="https://user-images.githubusercontent.com/25170843/161538698-8cebccef-7d40-4267-8325-6bd0049b3326.png">